### PR TITLE
builder/amazon-chroot: add mount_options configuration option

### PR DESCRIPTION
**Pull request by catsby**
Should fix https://github.com/mitchellh/packer/issues/1848

### CODE REVIEW

File: builder/amazon/chroot/step_mount_device.go [hunk 1]
```diff
@@ -23,7 +24,8 @@ type mountPathData struct {
 //   mount_path string - The location where the volume was mounted.
 //   mount_device_cleanup CleanupFunc - To perform early cleanup
 type StepMountDevice struct {
-	mountPath string
+	mountPath    string
+	MountOptions []string
```
Comment (mitchellh) [inline builder/amazon/chroot/step_mount_device.go]: Can you put this above `mountPath`? I like to put exported above unexported, example:

```
MountOptions []string

mountPath string
```

**Comment by mitchellh** [comment]
One nitpick but LGTM

**Comment by catsby** [comment]
Docs added in  24106de:


* `mount_options` (array of strings) – Options to supply the `mount` command
when mounting devices. Each option will be prefixed with `-o ` and supplied to
the `mount` command ran by Packer. Because this command is ran in a shell, user
discrestion is advised. See [this manual page for the mount command][1] for valid
file system specific options

[1]: http://linuxcommand.org/man_pages/mount8.html

**Comment by mitchellh** [comment]
LGTM!

**Pull request merged**